packages/crypto/tests/primitives/ed25519.spec.ts

```diff
@@ -315,17 +315,24 @@ describe('Ed25519', () => {
       expect(signature).to.be.instanceOf(Uint8Array);
     });
 
-    for (const vector of ed25519Sign.vectors) {
-      it(vector.description, async () => {
-        const signature = await Ed25519.sign({
-          key  : vector.input.key as Jwk,
-          data : Convert.hex(vector.input.data).toUint8Array()
-        });
-
-        const signatureHex = Convert.uint8Array(signature).toHex();
-        expect(signatureHex).to.deep.equal(vector.output);
+    describe('Web5TestVectorsCryptoEd25519', () => {
```

Comment:
all the other examples are 1 level deep:
```
describe('PresentationExchange', () -> describe('Web5TestVectorsPresentationExchange', () 
describe('Verifiable Credential Tests', () -> describe('Web5TestVectorsCredentials', ()
```

yours are 2 levels deep
```
describe('Ed25519', () -> describe('verify()', () -> describe('Web5TestVectorsCryptoEd25519', ()
```


I -think- it should work and get to the checkmark screen, but if it doesn't then this will probably be the reason.

I think we should put all vector test in one file eventually for better ease of use, but we can discuss that offline

Reply:
junit xml doesnt have a way to do more than one class as far as i know, so the mocha junit reporter thing will just smush it down, which may or may not pass the regex. Probably best to just keep doing it the way we've established works.

Reply:
Let's give it a shot and see how it does.

I had the same thought but the reason I ended up nesting them is that, otherwise, it isn't clear by looking at the tests that those methods are being rigorously tested.